packages/styles/src/components/tag/tag.stories.js

```diff
@@ -0,0 +1,24 @@
+export default {
+  title: 'Patterns/Badges, Chips, and Tags',
+  argTypes: {
+    variant: {
+      control: { type: 'select' },
+      options: ['', 'danger', 'success', 'warning'],
+      description: 'Adds a modifier class to the component to display different color variations (see generated HTML)'
+    },
+  },
+};
+
+
+const TagTemplate = (args) => {
+  const tagType = args.variant ? `cbp-tag--${args.variant}` : null;
+  
+  return `
+      <div class="${tagType ? 'cbp-tag ' + tagType : 'cbp-tag'}">
+        Hotlist
+      </div>
+  `;
+};
```

Comment:
- Destructure the `args` parameter
- Similar to the badges, we can leave the `cbp-tag` name in the `class` attribute and only hide/show the modifier class. We can do something like this `<div class="cbp-tag ${variant ? 'cbp-tag--' + variant : ''}">` since empty strings in JS are falsey and one of the options in the list.